stripe-crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/OnrampCoordinator.kt

```diff
@@ -126,16 +136,29 @@ class OnrampCoordinator @Inject internal constructor(
                 LinkController.create(
                     activity = activity,
                     presentPaymentMethodsCallback = { /* No-op for now */ },
-                    lookupConsumerCallback = { handleConsumerLookupResult(it) },
+                    lookupConsumerCallback = { lookupCallback(it) },
                     createPaymentMethodCallback = { /* No-op for now */ },
-                    authenticationCallback = { handleAuthenticationResult(it) },
-                    registerConsumerCallback = { handleRegisterNewUserResult(it) },
+                    authenticationCallback = { authCallback(it) },
+                    registerConsumerCallback = { registerCallback(it) },
                 )
             }
 
+            val cryptoApiRepository = CryptoApiRepository(
```

Comment:
Similar to the above, let's resolve the Coordinator via DI. See how LinkController.create does it [here](https://github.com/stripe/stripe-android/blob/master/paymentsheet/src/main/java/com/stripe/android/link/LinkController.kt#L535).